File: bryntum/gantt/examples/aspnet/BryntumGanttCrudDemo/app/lib/models/CalendarInterval.js
import { CalendarIntervalModel } from '../../node_modules/bryntum-gantt/gantt.module.js';

export default class CalendarInterval extends CalendarIntervalModel {
    static get fields() {
        return [
            { name : 'Id' },
            { name : 'recurrentStartDate', dataSource : 'RecurrentStartDate' },
            { name : 'recurrentEndDate', dataSource : 'RecurrentEndDate' },
            { name : 'startDate', dataSource : 'StartDate', type : 'date' },
            { name : 'endDate', dataSource : 'EndDate', type : 'date' },
            { name : 'isWorking', dataSource : 'IsWorking', defaultValue : false },
            { name : 'cls', dataSource : 'Cls' }
        ];
    }
}

CalendarInterval.idField = 'Id';

File: bryntum/gantt/examples/aspnetcore/BryntumGantt/Bryntum/Gantt/Entities/CalendarInterval.cs
using Bryntum.CRUD.Entities;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace Bryntum.Gantt
{
    public class CalendarInterval : General
    {
        public override int Id { get; set; }

        [JsonIgnore]
        public int CalendarIdRaw { get; set; }

        public string RecurrentStartDate { get; set; }

        public string RecurrentEndDate { get; set; }

        public DateTime? StartDate { get; set; }

        public DateTime? EndDate { get; set; }

        [JsonIgnore]
        public byte? IsWorkingRaw { get; set; }

        public bool IsWorking
        {
            get { return IsWorkingRaw == 1; }
            set { IsWorkingRaw = Convert.ToByte(value); }
        }

        public string Cls { get; set; }

        [JsonIgnore]
        public virtual Calendar Calendar { get; set; }
    }
}
